src/display/api.js

```diff
@@ -1199,9 +1200,11 @@ class PDFDocumentProxy {
  *   The default value is `AnnotationMode.ENABLE`.
  * @property {Array<any>} [transform] - Additional transform, applied just
  *   before viewport transform.
- * @property {Object} [canvasFactory] - The factory instance that will be used
- *   when creating canvases. The default value is {new DOMCanvasFactory()}.
- * @property {Object | string} [background] - Background to use for the canvas.
+ * @property {BaseCanvasFactory} [canvasFactory] - The factory instance that
+ *   will be used when creating canvases. The default value is
+ *   {new DOMCanvasFactory()}.
+ * @property {CanvasGradient | CanvasPattern | string} [background] - Background
+ *    to use for the canvas.
```

Comment:
Too much indentation on the second line.
```suggestion
 * @property {CanvasGradient | CanvasPattern | string} [background] - Background
 *   to use for the canvas.
```